src/makeCalculator.js

```diff
@@ -37,7 +37,47 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(a) {
+      calculator.result += a;
```

Comment:
```suggestion
      this.result += a;
```

You should use `this`